Errors when compiling it using Visual Studio 2019

I am getting the following errors when compiling it using Visual Studio 2019 via the build.bat:

imgview.cpp
c1xx: fatal error C1083: Cannot open source file: '..\imgview.cpp': No such file
 or directory

After fixing the buiild.bat:

imgview.cpp
imgview.cpp(810): error C2664: 'HANDLE CreateThread(LPSECURITY_ATTRIBUTES,SIZE_T
,LPTHREAD_START_ROUTINE,LPVOID,DWORD,LPDWORD)': cannot convert argument 3 from '
DWORD (__cdecl *)(void *)' to 'LPTHREAD_START_ROUTINE'
imgview.cpp(810): note: None of the functions with this name in scope match the
target type
C:\Program Files\Windows Kits\10\include\10.0.19041.0\um\processthreadsapi.h(170
): note: see declaration of 'CreateThread'

--- a/build.bat
+++ b/build.bat
@@ -1,6 +1,15 @@
 @echo off
 
+set cd=%~dp0
+set build=%cd%build
+pushd %cd%
+IF NOT EXIST build (mkdir build)
+pushd %build%
+
 cl /nologo /GS- /Gs9999999 /Gm- /EHa- /GF /Gy /GA /GR- /Zi /Fe:imgview.exe ..\imgview.cpp ^
 	/link /NODEFAULTLIB kernel32.lib user32.lib gdi32.lib ole32.lib shlwapi.lib shell32.lib advapi32.lib comctl32.lib uuid.lib gdiplus.lib ^
 	/subsystem:windows /OPT:REF /OPT:ICF /STACK:0x100000,0x100000 ^
   	&& imgview 
+
+popd
+popd

--- a/imgview.cpp
+++ b/imgview.cpp
@@ -15,6 +15,19 @@ using namespace Gdiplus::DllExports;
 
 /////////////////////////////////////////////////////////////////
 
+#ifdef __cplusplus 
+extern "C"
+#endif
+__declspec(naked) void _ftol2_sse()
+{
+	__asm
+	{
+		fistp dword ptr [esp-4]
+		mov   eax,[esp-4]
+		ret
+	}
+}
+
 extern "C" int _fltused = 0;
 typedef HRESULT (*GetDpiForMonitorType)(HMONITOR hmonitor, MONITOR_DPI_TYPE dpiType, UINT *dpiX, UINT *dpiY);
 typedef BOOL (*SetProcessDpiAwarenessContextType)(DPI_AWARENESS_CONTEXT value);
@@ -670,7 +683,7 @@ LRESULT CALLBACK FrameProcedure(HWND window, UINT message, WPARAM wParam, LPARAM
 	return 0;
 }
 
-DWORD GetFolderContents(void *_filePath) {
+DWORD WINAPI GetFolderContents(void *_filePath) {
 	wchar_t *filePath = (wchar_t *) _filePath;
 	wchar_t pathBuffer[MAX_PATH + 4];
 	

